crates/turbopack-core/src/issue/mod.rs

```diff
@@ -346,6 +347,20 @@ impl CapturedIssuesVc {
     pub async fn is_empty(self) -> Result<BoolVc> {
         Ok(BoolVc::cell(self.await?.is_empty()))
     }
+
+    #[turbo_tasks::function]
+    pub async fn has_fatal(self) -> Result<BoolVc> {
+        let mut has_fatal = false;
+
+        for issue in self.strongly_consistent().await?.iter() {
```

Comment:
`strongly_consistent` can only be used on operations. `self` is already resolved to a cell.

Only use strongly_consistent after calling a function.

The correct way to make this strongly consistent would be `issues_vc.has_fatal().strongly_consistent().await?`